### DISCUSSION TITLE
Slim down light.turn_on in 'visual editor'

### DISCUSSION THREAD

**Discussion by onionmad** [home-assistant/frontend#11400] [Automations / scripts / scenes]
Since we got the service light.turn_on to support the visual editor, most of my automations (when editing 'visually', are about 300 pages of scrolling, due to sequential light.turn_on actions. 
Since most of the settings related to this service is unused by most people, most of the time, does it really make sense to spam by listing all these settings?

At this point, **I would prefer to be able to set a default view option for the service light.turn_on, to force yaml-editing mode, for each of the actions relying on this service.** 

As you can see, a lot of screen space could be saved, making it far easier to get an overview of the actions and general flow of the automation/script being edited. 

Note: I generally use the visual editor, as most other services are using a more reasonable amount of screen area, and manually toggling yaml-mode for only the actions with light.turn_on is getting real old, real fast.

![image](https://user-images.githubusercontent.com/62025572/150630689-7f3ba978-e796-4175-b28d-9930ce216a33.png)

**Comment by spacegaier** [comment]
Somewhat related to this discussion: https://github.com/home-assistant/frontend/discussions/10064

Another thought: Having the option to collapse and expand automation sections.
